dist/profile/manifests/ldap.pp

```diff
@@ -241,7 +241,12 @@
     action => 'accept',
   }
 
-
+  firewall { '107 accept inbound LDAPS from nginx.azure.jenkins.io':
+    proto  => 'tcp',
+    source => 'nginx.azure.jenkins.io.',
```

Comment:
I thought the traffic pattern was that nginx.azure.jenkins.io was the public IP for ingress, is it also going to be the source IP for outbound packets?

I think this should refer to `beta.accounts.jenkins.io` regardless, referring to `nginx.azure.jenkins.io` here is "leaking" an implementation detail of how the application is wired together through nginx.azure.jenkins.io. By referring to `beta`accounts.jenkins.io` any changes we make in the future won't require a firewall update.